Updated API endpoints by adding the wellness check feature (#344)
gordon-cs/gordon-360-api: README.md

`api/news/new` Gets every student news entry that has been accepted, has not expired (as described above), and is new since 10am on the day before.

### Wellness Check

> Back endpoint responsible for fetching and sending information to the databse reguarding the answers to the wellness check.

Reviewer: "fixing typos"
```suggestion
Back endpoint responsible for fetching and sending information to the database regarding the answers to the wellness check.
```

##### GET 

> `api/wellness` Gets the latest answer a student has sent as well as a boolean that specifies weather the answer is still valid based on the current time and time submitted.  

Reviewer: ```suggestion
`api/wellness` Gets the latest answer a student has sent, as well as a boolean that specifies whether the answer is still valid based on when the answer was submitted.  
```

##### POST

> `api/wellness` Sends an answer boolean to the database that specifies whether a student is symptomatic or not, true =  Symptomatic, false = not symptomatic

Reviewer: ```suggestion
`api/wellness` Sends an answer boolean to the database that specifies whether a student is symptomatic or not: true = symptomatic, false = not symptomatic.
```
